indexer/services/l1/service.go

```diff
@@ -213,7 +229,7 @@ func (s *Service) Update(newHeader *types.Header) error {
 		return nil
 	}
 
-	if lowest.Hash != headers[0].ParentHash {
+	if lowest.Number > 0 && lowest.Hash != headers[0].ParentHash {
```

Comment:
Ideally this service code can be deduplicated between the `l1` and `l2` service, out of scope of this PR

Reply:
Perhaps using `op-service` longer term?